### PR TITLE
(Gabi will edit) mapping from prisma entities to domain #67

### PR DESCRIPTION
**Pull request by svallory**

### CODE REVIEW

**Review by antonio-marcos1989 — DISMISSED** [review]

**Comment by albnunes** [comment]
Do a rebase

File: apps/atqr/api/src/app/challenge.controller.ts [hunk 1]
```diff
@@ -134,20 +134,7 @@ export class ChallengeController {
...
-    @Body() updateCreditCardTokenDto: UpdateCreditCardTokenDto
-  ): void {
-    return;
-  }
-  @Patch(':id/status')
-  async updateStatus(
-    @Param('id') id: Guid,
-    @Body() status: ChallengeStatus
-  ): Promise<void> {
+   async updateStatus(id: Guid, status: ChallengeStatus): Promise<void> {
```
Comment (yfernandes) [inline apps/atqr/api/src/app/challenge.controller.ts:137]: Why were the method and controller decorators decorator deleted?

**Comment by yfernandes** [comment]
This PR has 37 modified files, spanning 3 projects, most changes have no connection with each other, and the PR has no description.

@albnunes and I are looking for a place to start reviewing for at least 30min. Please provide a clear description of your PR and clean your commit history.

**Review by yfernandes — CHANGES_REQUESTED** [review]
This PR has 37 modified files, spanning 3 projects, most changes have no connection with each other, and the PR has no description.

@albnunes and I are looking for a place to start reviewing for at least 30min. Please provide a clear description of your PR and clean your commit history. This PR is about mapping Prisma entities to domain, and it should be reserved for that!

**Pull request closed without merging**